src/js/shot-generator/Character.js

```diff
@@ -72,37 +74,62 @@ const characterFactory = data => {
   let mesh
   let skeleton
   let armatures
-
+  let parentRotation = new THREE.Quaternion()
+  let parentPosition = new THREE.Quaternion()
   mesh = data.scene.children.find(child => child instanceof THREE.SkinnedMesh) ||
          data.scene.children[0].children.find(child => child instanceof THREE.SkinnedMesh)
 
   armatures = data.scene.children[0].children.filter(child => child instanceof THREE.Bone)
-
+  if (armatures.length === 0 ) {  // facebook export is different
+    //console.log('searching in: ',data.scene.children[0].children[0].children)
+    armatures = data.scene.children[0].children[0].children.filter(child => child instanceof THREE.Bone)
+    for (var bone of armatures)
+    {
+      //console.log('this scale: ', data.scene.children[0].children[0])
+      //bone.scale.multiply(data.scene.children[0].children[0].scale)
+      bone.scale.set(1,1,1)
+      bone.quaternion.multiply(data.scene.children[0].children[0].quaternion)
+      //bone.position.multiply(data.scene.children[0].children[0].scale)
+      bone.position.set(bone.position.x,bone.position.z,bone.position.y)
```

Comment:
Not sure what this line does. Seems to re-set to its current value...

Reply:
bone.position.set(bone.position.x,bone.position.z,bone.position.y)

it switches z and y positions, because the mesh comes in rotated 90deg